lib/Dialect/Torch/Transforms/Passes.cpp

```diff
@@ -86,6 +91,36 @@ void mlir::torch::Torch::createTorchFunctionToTorchBackendPipeline(
       options.backendLegalOps, options.extraLibrary));
 }
 
+static void createOnnxShapeRefinementPipeline(
```

Comment:
We should either have this as an actual pass that runs iteratively, or we should just put those two in the other pipeline as-is. 

For the sake of getting this in sooner rather than later, let's just remove this function and add the two passes to the onnx-to-torch-backend-pipeline.